java/app/src/main/res/values/hosts.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="aws_host">http://reactshoppe.s3-website-us-east-1.amazonaws.com</string>
+    <string name="aws_host">http://reactshoppe.s3-website-us-east-1.amazonaws.com/data/products.json</string>
```

Comment:
technically `aws_host` with the S3 domain seems more appropriate than `aws_host` with the full path to the `products` endpoint. Would it be a good idea to either:

a. rename this config to `products_endpoint` or
b. keep the `aws_host` and concatenate with the product endpoint URL in the code

Reply:
resolved using a.